hivesim-ts/simulators/ts-sim-demo/main.ts

```diff
@@ -0,0 +1,162 @@
+import exp from "constants";
```

Comment:
Sure, technically this is doable, but I'm not sure that it would feel cleaner, especially as more tests are introduce.  I do have the Dockerfile in  `/simulators`, which allows the suite to run using the same hive CLI commands as others, but it felt simpler to keep all `nodejs` code in one directory.

Another approach could be to pubish any typescript suites in their own github repo, and build the docker image from that instead.  

That would actually make it a little easier to add to existing test suites without needing to merge into the portal-hive repo itself.

Reply:
I am fine with "pubish any typescript suites in their own github repo". As that would keep hivesim and simulator code seperate.

It would be interesting what @ogenev thinks as he probably has a few helpful insights. Maybe ensuring this seperation isn't as important as I think it is and the PR is fine in that regard? So @ogenev can give the presendent for that.